content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -124,38 +130,36 @@ into the plots file `evaluation/plots/prc.json`:
 
 Similarly, it writes arrays for the
 [roc_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html)
-into `evaluation/plots/sklearn/roc.json`,
+and
 [confusion matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)
-into `evaluation/plots/sklearn/confusion_matrix.json`, and an image
-`evaluation/plots/importance.png` with a feature importance bar chart for
-additional plots.
-
-> DVC doesn't force you to use any specific file names, nor does it enforce a
-> format or structure of a metrics or plots file. It's completely
-> user/case-defined. Refer to `dvc metrics` and `dvc plots` for more details.
-
-You can view tracked metrics and plots with DVC. Let's start with the metrics:
+into JSON files in the `eval/live/plots` directory, and an image
+`eval/importance.png` with a feature importance bar chart for additional plots.
 
-```cli
-$ dvc metrics show
-Path             avg_prec    roc_auc
-evaluation.json  0.89668     0.92729
-```
+To view plots, first configure the axes and other specifications of your plots
+by adding a `plots` section to your `dvc.yaml`:
 
-To view plots, first specify which arrays to use as the plot axes. We only need
-to do this once, and DVC will save our plot configurations.
-
-```cli
-$ dvc plots modify evaluation/plots/prc.json -x recall -y precision
-Modifying stage 'evaluate' in 'dvc.yaml'
-$ dvc plots modify evaluation/plots/sklearn/roc.json -x fpr -y tpr
-Modifying stage 'evaluate' in 'dvc.yaml'
-$ dvc plots modify evaluation/plots/sklearn/confusion_matrix.json \
-    -x actual -y predicted -t confusion
-Modifying stage 'evaluate' in 'dvc.yaml'
+```yaml
+plots:
+  - eval/importance.png
+  - Precision-Recall:
+      x: recall
+      y:
+        eval/prc/train.json: precision
+        eval/prc/test.json: precision
+  - ROC:
+      x: fpr
+      y:
+        eval/live/plots/sklearn/roc/train.json: tpr
+        eval/live/plots/sklearn/roc/test.json: tpr
+  - Confusion-Matrix:
+      template: confusion
+      x: actual
+      y:
+        eval/live/plots/sklearn/cm/train.json: predicted
+        eval/live/plots/sklearn/cm/test.json: predicted
 ```
 
-Now let's view the plots. You can run `dvc plots show` on you terminal (shown
+Now let's view the plots. You can run `dvc plots show` on your terminal (shown
 below), which generates an HTML file you can open in a browser. Or you can load
```

Comment:
So should the section be called **Render plots** or **Configure and render plots**?

Also, 💅🏼 
```suggestion
To render them, you can run `dvc plots show` (shown below),
which generates an HTML file you can open in a browser. Or you can load
```